src/lib/components/chat/ChatIntroduction.svelte

```diff
@@ -40,14 +43,18 @@
 		</div>
 	</div>
 	<div class="lg:col-span-2 lg:pl-24">
-		<AnnouncementBanner classNames="mb-4" title="Chat UI is now open sourced on GitHub">
-			<a
-				target="_blank"
-				href="https://github.com/huggingface/chat-ui"
-				class="mr-2 flex items-center underline hover:no-underline"
-				><CarbonArrowUpRight class="mr-1" /> GitHub repo</a
-			>
-		</AnnouncementBanner>
+		{#if announcementBanners.length > 0}
```

Comment:
There is no need for the `#if` here, the `#each` is enough.

Reply:
Ok thanks I'll remove the if.